modelserver-jsonforms-property-view/src/browser/modelserver-widget-provider.ts

```diff
@@ -56,21 +56,6 @@ export abstract class ModelserverAwareWidgetProvider extends JsonFormsPropertyVi
 
     protected abstract handleChanges(jsonFormsData: Object | undefined): void;
 
-    /** Returns a string representing the model URI relative to the workspace root URI.
-     * @param sourceUri a non-encoded URI string
-     */
-    protected getRelativeModelUri(sourceUri: string): string {
-        const workspaceUri = this.workspaceService.getWorkspaceRootUri(new URI(sourceUri));
```

Comment:
The `workspaceService` instance variable was only used by this method that is now removed. Should `workspaceService` also be deleted?

Reply:
Yes, totally, thanks for catching that!